code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -112,6 +112,23 @@
 	flags_inv = HIDEGLOVES|HIDESHOES|HIDEJUMPSUIT
 	resistance_flags = NONE
 
+/obj/item/clothing/head/syndicatefake/nuke

```

Comment:
this needs to be moved to https://github.com/yogstation13/Yogstation-TG/blob/195c228bb1c58602884cfaab3351c318f097673b/code/modules/clothing/head/misc.dm#L61

Reply:
you mean "flags_inv = HIDEGLOVES|HIDESHOES|HIDEJUMPSUIT
	resistance_flags = NONE" moved under the fake syndie space helmet?

Reply:
Added

Reply:
What I mean is, that this entire "paragraph" of code relating to the helmet needs to be moved to this file, to below the other replica helmet